Client/Frontend/Browser/TabManager.swift

```diff
@@ -453,14 +453,39 @@ class TabManager: NSObject {
                     for delegate in self.delegates {
                         delegate.get()?.tabManagerDidAddTabs(self)
                     }
+                } else {
+                    self.showFocusPromoToast()
                 }
                 self.eraseUndoCache()
             })
         }
 
         delegates.forEach { $0.get()?.tabManagerDidRemoveAllTabs(self, toast: toast) }
     }
-    
+
+    func showFocusPromoToast() {
+        // add check for event stuff
+        guard let shouldShowFocusUI = LeanplumIntegration.sharedInstance.getBoolVariableFromServer(key: "shouldShowFocusUI"), let focus = NSURL(string: "firefox-focus://"), let klar = NSURL(string: "firefox-klar://"), !UIApplication.shared.canOpenURL(focus as URL) && !UIApplication.shared.canOpenURL(klar as URL) && shouldShowFocusUI else {
```

Comment:
Not sure why it should move to the LeanplumIntegration @farhanpatel ? I think that should serve as a wrapper for the LP SDK only, so not sure why this logic (which concerns the use of LP) should move there? I don't think it will used by other users of LeanplumIntegration so no benefit putting it there. Overall though, not a huge deal. :)